<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumericQuestionWidgetClass</class>
 <widget class="QWidget" name="NumericQuestionWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NumericQuestionWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame{
border-radius:10px ;
background-color: rgba(0,0,0,0.4);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Question</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="questionStatement">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>here comes the text of the numeric question</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="answerLineEdit">
        <property name="placeholderText">
         <string>Your answer comes here</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="approximateAnswer">
        <property name="text">
         <string>Approximate answer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="givePossibleAnswers">
        <property name="text">
         <string>Give possible options</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendAnswer">
        <property name="text">
         <string>Send answer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="timerProgressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
